src/main/java/com/Acrobot/ChestShop/Commands/ItemInfo.java

```diff
@@ -40,30 +40,9 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
             item = parseEvent.getItem();
         }
 
-        if (MaterialUtil.isEmpty(item)) {
-            return false;
-        }
-
-        iteminfo.send(sender);
-        try {
-            iteminfo_fullname.send(sender, "item", MaterialUtil.getName(item));
-        } catch (IllegalArgumentException e) {
-            sender.sendMessage(ChatColor.RED + "Error while generating full name. Please contact an admin or take a look at the console/log!");
-            ChestShop.getPlugin().getLogger().log(Level.SEVERE, "Error while generating full item name", e);
-            return true;
-        }
-
-        try {
-            iteminfo_shopname.send(sender, "item", MaterialUtil.getSignName(item));
-        } catch (IllegalArgumentException e) {
-            sender.sendMessage(ChatColor.RED + "Error while generating shop sign name. Please contact an admin or take a look at the console/log!");
-            ChestShop.getPlugin().getLogger().log(Level.SEVERE, "Error while generating shop sign item name", e);
-            return true;
-        }
-
         ItemInfoEvent event = new ItemInfoEvent(sender, item);
         ChestShop.callEvent(event);
 
-        return true;
+        return !MaterialUtil.isEmpty(item);
```

Comment:
Are you sure any other event listener doesn't throw an error when the item is null/empty? (As you moved the check from before calling the event to after it and don't check that again anywhere)